lib/ads/interstitial_ad_manager.js

```diff
@@ -192,6 +192,13 @@ shaka.ads.InterstitialAdManager = class {
    * @param {shaka.extern.Interstitial} interstitial
    */
   async addMetadata(interstitial) {
+    if (this.basePlayer_.getLoadMode() == shaka.Player.LoadMode.SRC_EQUALS &&
+        this.usingBaseVideo_) {
+      shaka.log.alwaysWarn(
+          'Unsupported interstitial when using single media element',
```

Comment:
This is necessary because our system to only use one video depends on the preload api and there is no fallback for src=